Stream_HomeWork_10/src/Person.java

```diff
@@ -0,0 +1,22 @@
+
+public class Person {
+
```

Comment:
You already have a Person class and it is exactly the same as another one. Why do you need the second one? Do not repeat yourself!